soroban-env-host/src/host.rs

```diff
@@ -2254,7 +2265,12 @@ impl VmCallerEnv for Host {
         let scv = self.visit_obj(b, |hv: &ScBytes| {
             self.metered_from_xdr::<ScVal>(hv.as_slice())
         })?;
-        if Val::can_represent_scval(&scv) {
+        // Metering bug: the representation check is not metered,
```

Comment:
Could you file a bug on this for future consolidation?

Reply:
Opened https://github.com/stellar/rs-soroban-env/issues/1318